ldtp/__init__.py

```diff
@@ -414,8 +415,8 @@ def waittillguinotexist(window_name, object_name = '',
                                        guiTimeOut)
 def guiexist(window_name, object_name = ''):
     return _remote_guiexist(window_name, object_name)
-def launchapp(cmd, args = [], delay = 0, env = 1, lang = "C"):
-    return _remote_launchapp(cmd, args, delay, env, lang)
+def launchapp(cmd, args = [], delay = 0, env = 1, lang = "C", logfiles = None ):
+    return _remote_launchapp(cmd, args, delay, env, lang, logfiles)
 def hasstate(window_name, object_name, state, guiTimeOut = 0):
     return _remote_hasstate(window_name, object_name, state, guiTimeOut)
 def selectrow(window_name, object_name, row_text):
```

Comment:
On line 449, there is a unicode() call that should be converted to str() for python3